ostd/src/arch/x86/mod.rs

```diff
@@ -129,7 +129,6 @@ pub(crate) unsafe fn init_on_ap() {
 
 pub(crate) fn interrupts_ack(irq_number: usize) {
     if !cpu::CpuException::is_cpu_exception(irq_number as u16) {
-        kernel::pic::ack();
```

Comment:
Removing this Line is OK, and all interrupts require `apic.eoi()` to ack interrupts. Also, we should find another way to trigger `apic. eoi()` when we support more device drivers (not in this PR).